include/deal.II/base/config.h.in

```diff
@@ -46,6 +46,7 @@
 #cmakedefine DEAL_II_WITH_GSL
 #cmakedefine DEAL_II_WITH_HDF5
 #cmakedefine DEAL_II_WITH_KOKKOS
+#cmakedefine DEAL_II_WITH_KOKKOS_BACKEND
```

Comment:
If you don't tie it into the feature configuration (yet) what about you (temporarily) call the variable `DEAL_II_USE_KOKKOS_BACKEND` for now?

I suggest we revisit the whole kokkos configuration once the bundled version is in and/or prior to the next release.